src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -245,6 +245,13 @@ limitations under the License.
     <attribute side="server" code="0x4033" define="NULLABLE_GLOBAL_ENUM" type="TestGlobalEnum" isNullable="true" writable="true">nullable_global_enum</attribute>
     <attribute side="server" code="0x4034" define="NULLABLE_GLOBAL_STRUCT" type="TestGlobalStruct" isNullable="true" writable="true">nullable_global_struct</attribute>
 
+    <!-- These attributes are intended to return internal failures when read.
+         They are internal SDK-test code that are intended to only run trhrough EMBER interfaces instead of
+         code-backed support (to be able to test ember-based error paths)
+      -->
+    <attribute side="server" code="0x4100" define="INT8U" type="int8u" writable="true" min="1" max="0xcb" default="1" optional="true">readFailureCode</attribute>
+    <attribute side="server" code="0x4101" define="INT32U" type="int32u" writable="true" default="0" optional="true">failureInt32U</attribute>
```

Comment:
Please don't use ids in the 0x4... range for these things.  We've been trying to use that for nullable versions of attributes with lower-numbered ids.

Could put the new thing before the nullable attributes, or just start with 0x6 or something.

Reply:
My first pick was the next of 0x5000 and it turns out that the range has to be lower than 0x4FFF ...

so I must be between 0 and 0x4FFFF ... I could use 1,2 or 3 I guess.

Reply:
will switch to 0x3000 as a base.

Reply:
Updated to 0x3000/0x30001